data/json/monsters/bird.json

```diff
@@ -163,6 +167,7 @@
   },
   {
     "id": "mon_cockatrice",
+    "looks_like": "mon_chicken",
```

Comment:
```suggestion
```

Reply:
It is defined already on line 185.

Reply:
To keep the formatting consistent, I shall remove the line 185 ``looks_like``.